app/components/index_title_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<header class="documentHeader">
```

Comment:
A new view component to show document titles on the search result page (a replacement for app/views/catalog/_index_header_default.html.erb)